src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ connector-iaas-tag.key = proactive-connector-iaas
 connector-iaas-tag.value = rZa/Qo,2eh+Fn5c!iqI9*ixSh7:K8x
 infrastructure-id-tag.key = proactive-infrastructure-id
 connector-iaas.vm-user-login = admin
+# server.port=8080
```

Comment:
Why adding this comment?

Reply:
beacuse it took me around 1 hour to figure where to change the server, saving the next person this one hour ;)

Reply:
So at least add another comment before to explain. Exp:
`#Uncomment this to set the server port`

Reply:
noted